libs/newsletter/src/actions/newsletter.actions.ts

```diff
@@ -0,0 +1,46 @@
+import { Action } from '@ngrx/store';
+
+import { DaffNewsletter } from '../models/newsletter.model';
+
+export enum DaffNewsletterActionTypes {
+  NewsletterSubscribeAction = "[Daff-Newsletter] Newsletter Subscribe Action",
+  NewsletterCancelAction = "[Daff-Newsletter] Newsletter Cancel Action",
+  NewsletterSuccessSubscribeAction = "[Daff-Newsletter] Succeeded on Newsletter Subscribe Action",
+  NewsletterFailedSubscribeAction = "[Daff-Newsletter] Failed on Newsletter Subscribe Action",
+  NewsletterRetry = "[Daff-Newsletter] Retrying to cancel newsletter",
```

Comment:
Clarify this action text: `[Daff-Newsletter] Retrying to cancel newsletter` it should probably be:

```
[Daff-Newsletter] Retry submission
```